# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/clang/stdcxxabi.patch.musl
# Copyright (C) 2021 - 2022 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- clang-15.0.5.src/lib/Driver/ToolChain.cpp.vanilla	2022-12-26 11:10:45.711204955 +0100
+++ clang-15.0.5.src/lib/Driver/ToolChain.cpp	2022-12-26 11:11:49.100051625 +0100
@@ -1012,7 +1012,7 @@
 
   switch (Type) {
   case ToolChain::CST_Libcxx:
-    CmdArgs.push_back("-lc++");
+    CmdArgs.push_back("-lc++"); CmdArgs.push_back("-lc++abi");
     if (Args.hasArg(options::OPT_fexperimental_library))
       CmdArgs.push_back("-lc++experimental");
     break;
